node-operators/networks/compile-binary.md

```diff
@@ -30,24 +30,25 @@ Let's check out the latest release:
 git checkout tags/$(git tag | tail -1)
 ```
 
-Next, install a Substrate development environment, including Rust, by executing:
+Next, install Rust and its prerequisites [via Rust's recommended method](https://www.rust-lang.org/tools/install){target=_blank} by executing:
 
 ```
---8<-- 'code/setting-up-node/substrate.md'
+--8<-- 'code/setting-up-node/installrust.md'
 ```
 
-Lastly, build parachain binary:
+Next, update your PATH environment variable by running:
 
 ```
-cargo build --release
+--8<-- 'code/setting-up-node/updatepath.md'
 ```
 
-![Compiling Binary](/images/fullnode/compile-binary1.png)
-
-If a _cargo not found error_ shows up in the terminal, manually add Rust to your system path or restart your system:
```

Comment:
I sort of liked this error message because let's say I installed Rust but then the next day I tried this and I get the `cargo not found` error.